fabric/src/main/java/org/mcaccess/minecraftaccess/fabric/MainClassFabric.java

```diff
@@ -1,12 +1,16 @@
 package org.mcaccess.minecraftaccess.fabric;
 
+import net.blay09.mods.balm.api.Balm;
+import net.blay09.mods.balm.api.EmptyLoadContext;
+import net.blay09.mods.balm.api.client.BalmClient;
 import net.fabricmc.api.ModInitializer;
 
 import org.mcaccess.minecraftaccess.MainClass;
 
 public class MainClassFabric implements ModInitializer {
     @Override
     public void onInitialize() {
-        MainClass.init();
+        Balm.initializeMod(MainClass.MOD_ID, EmptyLoadContext.INSTANCE, () -> {});
```

Comment:
The documentation only says you need the client initialiser in the client entrypoint, this common initialiser appears to do absolutely nothing. Same with NeoForge.

https://balm.twelveiterations.com/docs/concepts/entrypoints#client-entrypoint-1